docs/setup-robusta/node-selector.rst

```diff
@@ -21,11 +21,38 @@ Add the following to your Helm values:
       nodeSelector:
         kubernetes.io/os: linux
 
+    # if using Robusta's embedded kube-prometheus-stack, you can configure the Prometheus Operator's components to run on a specific node too
+    kube-prometheus-stack:
```

Comment:
I suppose we only need one of these, no?

Reply:
@pavangudiwada I think they're all needed. Each subvalue here controls a different component in the Prometheus stack. You need to add the selector to each one.